### PR TITLE
libglvnd: switch to meson toolchain

### PR DESCRIPTION
**Pull request by SupervisedThinking**
Switch to meson build & disable OpenGL ES1 & ES2 if it's not enabled.

### CODE REVIEW

File: packages/graphics/libglvnd/package.mk [hunk 1]
```diff
@@ -9,10 +9,11 @@ PKG_SITE="https://github.com/NVIDIA/libglvnd"
 PKG_URL="https://github.com/NVIDIA/libglvnd/archive/v${PKG_VERSION}.tar.gz"
 PKG_DEPENDS_TARGET="toolchain libX11 libXext xorgproto"
 PKG_LONGDESC="libglvnd is a vendor-neutral dispatch layer for arbitrating OpenGL API calls between multiple vendors."
-PKG_TOOLCHAIN="autotools"
+PKG_TOOLCHAIN="meson"
```
Comment (heitbaum) [inline packages/graphics/libglvnd/package.mk:12]: Please remove PKG_TOOLCHAIN and allow auto-detect

**Review by heitbaum — CHANGES_REQUESTED** [review]

**Comment by heitbaum** [comment]
This commit supersedes #5039

File: packages/graphics/libglvnd/package.mk (hunk 1 above)
Comment (SupervisedThinking) [reply packages/graphics/libglvnd/package.mk:12]: indeed, can be dropped. I'll push an update later

**Comment by SupervisedThinking** [comment]
https://github.com/LibreELEC/LibreELEC.tv/pull/5039 was probably rage quitted anyway 🤔

**Review by heitbaum — APPROVED** [review]

**Review by CvH — APPROVED** [review]

**Pull request merged**